erpnext/accounts/doctype/loyalty_program/test_loyalty_program.py

```diff
@@ -38,6 +39,7 @@ def test_loyalty_points_earned_single_tier(self):
 		)
 
 		self.assertEqual(si_original.get("loyalty_program"), customer.loyalty_program)
+		self.assertEqual(lpe.get("loyalty_program_tier"), "Bronce")  # is always in the first tier
```

Comment:
2: this assertion should have been `None` because the tier's min spent is not reached.